#!/usr/bin/env python
PACKAGE = "kalman_filter"

from dynamic_reconfigure.parameter_generator_catkin import *
from kalman_filter.msg import KalmanFilterPosVelAccConf
from math import pi

gen = ParameterGenerator()

gen.add("kalmanFilterFlag", bool_t, KalmanFilterPosVelAccConf.RECONFIGURE_KALMAN_FILTER_FLAG, "Kalman Filter Flag", False)
gen.add("inputSigma", double_t, KalmanFilterPosVelAccConf.RECONFIGURE_INPUT_SIGMA, "Input Sigma", 0.04, 0, 1)
gen.add("measureSigma", double_t, KalmanFilterPosVelAccConf.RECONFIGURE_MEASURE_SIGMA, "Measure Sigma", 0.04, 0, 1)

exit(gen.generate(PACKAGE, "kalman_filter", "KalmanFilterPosVelAcc"))


